.github/workflows/push-test.yml

```diff
@@ -38,6 +38,12 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install -y libgtk-3-dev webkit2gtk-4.0 libappindicator3-dev librsvg2-dev patchelf
+      name: Disable self-updater
```

Comment:
Missed the dash (`-`) here ^^
```suggestion
    - name: Disable self-updater
```

Reply:
i mean i wasnt completely done but whatever, yeah